third_party/amd/lib/AMDGPUToLLVM/AMDGPUToLLVMPass.cpp

```diff
@@ -0,0 +1,79 @@
+#include "AMDGPUToLLVM/AMDGPUToLLVMPass.h"
+#include "Dialect/AMDGPU/IR/Dialect.h"
+#include "mlir/Dialect/LLVMIR/LLVMDialect.h"
+#include "mlir/IR/PatternMatch.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Support/LogicalResult.h"
+#include "mlir/Transforms/GreedyPatternRewriteDriver.h"
+#include "triton/Analysis/AxisInfo.h"
+#include "triton/Conversion/TritonGPUToLLVM/TypeConverter.h"
+// clang-format off
+#include "triton/Dialect/TritonGPU/Transforms/Utility.h"
+#include "amd/lib/TritonAMDGPUToLLVM/Utility.h"
+// clang-format on
+
+using namespace mlir;
+using namespace mlir::triton;
+using namespace mlir::triton::gpu;
```

Comment:
Not a fan of just `using` all namespaces. Can we drop all except the first two? We can consider add them if necessary later. But don't want to just enable all from the get-go.